app/models/gem_download.rb

```diff
@@ -92,6 +78,11 @@ def bulk_update(ary)
 
     private
 
+    def count_for(rubygem_id: 0, version_id: 0)
+      download = GemDownload.select(:count).find_by(rubygem_id: rubygem_id, version_id: version_id)
```

Comment:
This looks kinda weird. Can we please use ` GemDownload.where(rubygem_id: 0, version_id: 0).pluck(:count).first`?